ansible_navigator/actions/inventory.py

```diff
@@ -440,7 +440,7 @@ def _collect_inventory_details(
         self,
     ) -> Tuple[Union[None, str], Union[None, str], Union[None, int]]:
 
-        # pylint:disable=too-many-branches
+        # pylint:disable=too-many-branches,too-many-statements
```

Comment:
The linters are there to prevent making the project unmaintainable. Please don't add ignores — they are supposed to be used as a last resort when it's impossible to refactor the code properly.

Reply:
Too bad that wasn't enforced when the first person added too many branches and the `pylint:disable=too-many-branches` . . .

Reply:
And that's another problem. Sometimes it's okay to ignore things *temporarily* if you're going to be responsible for fixing that ASAP in a follow-up PR. But since not many people track such changes and forget about them, it's easier to just not introduce the linting violations in the initial patch.